Gemfile

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+
+gem 'cocoapods', '~> 1.15'
```

Comment:
I don't see any call to `bundle exec pod [command]` in this diff.

Does adding CocoaPods to the `Gemfile` results in the tools that need it (I assume some of the React Native toolchain) calling CocoaPods through Bundler?

Reply:
Correct, the React Native toolchain just needs CocoaPods installed and then it runs `pod install` on its own.

Reply:
Why do we need React Native? It's not used by this app? Or are we just taking advantage of some tooling?

Reply:
Ah, we just meant the Capacitor cross-platform tooling. 🙂